api/spec/integration/api/v2/platform/store_credits_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'swagger_helper'
+
+describe 'Store Credits API', swagger: true do
+  include_context 'Platform API v2'
+
+  resource_name = 'Store Credit'
+  options = {
+    include_example: 'user,created_by,category,credit_type',
+    filter_examples: [{ name: 'filter[user_id_eq]', example: 5 },
```

Comment:
```suggestion
    filter_examples: [{ name: 'filter[user_id_eq]', example: '5' },
```

Let's always use strings for IDs to not make a false impression that IDs will be always integers, in Vendo we use UUIDs

Reply:
@aleksandarpetrushev we also need to here add these attributes for ransack filterin